### PR TITLE
Pin semantic-release and add .gitattributes for WSL

### PR DESCRIPTION
**Pull request by luthermonson**
## Describe the Change

- Adds a pin for semantic-release
- Adds .gitattributes entires for *.sh to always have eol LF which helps make things work nicer on Windows using make in WSL.

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1298#" title="Author self-approved">luthermonson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: Makefile
Comment (karl-cardenas-coding) [inline Makefile:47]: @luthermonson this is not needed as the container is now built with all required nodejs dependencies.

Comment (luthermonson) [reply Makefile:47]: it is needed if i ever want to update the npm dependencies myself. it's just a nice to have and this is easily ran by hand if necessary. i can drop it if you dont want it

Comment (karl-cardenas-coding) [reply Makefile:47]: @luthermonson Yeah, please drop. We can control the dependencies and we would reject any changes that include changes to the dependencies.

**Comment by luthermonson** [comment]
@karl-cardenas-coding all fixed up

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**
